Content.Shared/Chemistry/ReactiveSystem.cs

```diff
@@ -38,6 +38,10 @@ public void ReactionEntity(EntityUid uid, ReactionMethod method, ReagentPrototyp
         if (!TryComp(uid, out ReactiveComponent? reactive))
             return;
 
+        // custom event for bypassing reactivecomponent stuff
```

Comment:
This is unclear why, please be clearer when doing weird lil' hacks :>

Reply:
@EmoGarbage404 what can i write here? I still don't 100% understand why it was required there...